Gemfile

```diff
@@ -9,6 +9,9 @@ gem 'bump', '~> 0.5'
 
 group :development do
   gem 'pry-byebug'
+  gem 'rubocop', '~>1.1'
+  gem 'rubocop-performance'
+  gem 'rubocop-rails'
```

Comment:
Add dependencies
As mentioned above `rubocop-rails` can be removed. Please let me know if you
would like to do so